parsers/FR_O.py

```diff
@@ -230,17 +238,18 @@ def get_date_name(zone_key: None, target_datetime: None):
 # Non-thermal sources names in api
 sources_mapping = {
     'biomass': ['bioenergies_mwh', 'bioenergies'],
-    'hydro': ['hydraulique_mwh', 'hydraulique'],
-    'solar': ['photovoltaique_mwh', 'photovoltaique', 'photovoltaique_avec_stockage'],
+    'hydro': ['hydraulique_mwh', 'hydraulique', 'micro_hydro'],
+    'solar': ['photovoltaique_mwh', 'photovoltaique', 'photovoltaique_avec_stockage', 'photovoltaique0'],
     'wind': ['eolien_mwh', 'eolien'],
     'geothermal': ['geothermie_mwh'],
-    'gas': ['biogaz']
+    'gas': ['biogaz'],
+    'oil': ['moteur_diesel']
 }
 
 
 def fetch_production(zone_key=None, session=None, target_datetime=None,
                      logger=logging.getLogger(__name__)):
-    if (target_datetime is None) and (zone_key != 'RE'):
+    if (target_datetime is None) and (zone_key != 'RE' or 'FR-COR'):
```

Comment:
This check is currently failing, so I'd suggest the following instead:
```suggestion
    if (target_datetime is None) and zone_key not in REAL_TIME_SOURCES.keys():
```